src/main/java/seedu/address/logic/relationship/EditRelationshipCommand.java

```diff
@@ -77,11 +77,11 @@ public CommandResult execute(Model model) throws CommandException {
         if (fullOriginUuid == fullTargetUuid) {
             throw new CommandException("Relationships must be between 2 different people");
         }
+        if (oldRelationshipDescriptor.equals(newRelationshipDescriptor)) {
```

Comment:
This check can be deleted as it is actually already performed in relationshipUtil